<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameSelectionPane</class>
 <widget class="QWidget" name="FrameSelectionPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current Frame: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="flow">
      <enum>QListView::Flow::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="DeleteFrame">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(68, 69, 80)</string>
       </property>
       <property name="text">
        <string>Delete Frame</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditDelete"/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="AddFrame">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(68, 69, 80)</string>
       </property>
       <property name="text">
        <string>Add Frame</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ListAdd"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
